resources/views/manage/tag/edit.blade.php

```diff
@@ -0,0 +1,17 @@
+<x-layout-manage>
+    <h2 class="font-bold mb-4 text-2xl">Tag</h2>
+
+    <form action="{{ route('tag.update', $tag) }}" method="POST">
+        @csrf
+        @method('PUT')
+
+        <fieldset class="mb-4">
+            <label for="name">Name:</label>
+            <input type="text" name="name" id="name" value="{{ old('name') ?? $tag->name }}"
```

Comment:
`old('name', $tag->name)` would be better in this case

Reply:
Didn't new this was possible, thank you